### DISCUSSION TITLE
guide/genswagger

### DISCUSSION THREAD

**Discussion by giscus** [jzero-io/jzero#57] [Announcements]
# guide/genswagger

说明 同时支持基于 go-zero api 文件和 protobuf 文件自动生成 swagger 文档 其中自动生成 swagger 的插件如下: goctl-swagger protoc-gen-openapiv2 由于 go-zero 官方仓库插件 goctl-swagger 不怎么维护了, 并在实际使用中存在一些 bug, 所以 fork 了一...

https://jzero.jaronnie.com/guide/genswagger.html

**Comment by shchzh85** [comment]
如果直接在上面这个页面里面进行测试，要填写登录后的auth值，在swagger文档中哪里可以填写呢，或是存储的

**Reply by jaronnie** [reply]
基于 proto 生成的 swagger吗

**Reply by shchzh85** [reply]
不是，
![2025-04-25_14-30-24](https://github.com/user-attachments/assets/79e208e2-0f57-43da-ba99-d7bf638e1f43)
看看这个图片，如果要填写head中的auth值，有一个锁，或是全局里面有一个authorize

**Reply by jaronnie** [reply]
加我微信  jaronnie
